spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -204,15 +201,15 @@ public Blob createBlob() throws StorageException {
 	 * such as if the bucket already exists
 	 */
 	public Bucket createBucket() {
-		return this.storage.create(BucketInfo.newBuilder(this.getBucketName()).build());
+		return this.storage.create(BucketInfo.newBuilder(this.location.getBucketName()).build());
```

Comment:
Did we somehow lose the `GoogleStorageResource.getBucketName()`?

Reply:
Done. Yeah this was a mistake, I added it back in and also modified the test to call this method.